### DISCUSSION TITLE
Text color issue in Composer view quoted message

### DISCUSSION THREAD

**Discussion by ramanareddy463** [GetStream/stream-chat-android#4662] [General]
Text color issue in Composer view quoted message.
for more information, check below screenshot.
![Screenshot 2023-02-13 133029](https://user-images.githubusercontent.com/8695932/218402349-d096a389-c31f-4148-9e2c-3acb64ef55f3.png)

Note: I am not using dark theme. I changed component color.

**Comment by MarinTolic** [comment]
Hi @ramanareddy463 ,

Could you please provide us with the full list of the changes you made that resulted in the component rendering in the manner you've shown.

Thank you.

**Comment by ramanareddy463** [comment]
Thanks for the reply @MarinTolic 

I have just changed the component colors.


```

   <io.getstream.chat.android.ui.message.composer.MessageComposerView
            android:id="@+id/messageComposerView"
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:background="@color/black"
            app:streamUiMessageComposerMessageInputTextColor="@color/white"
            app:streamUiMessageComposerDividerBackgroundDrawable="@color/black"
            app:streamUiMessageComposerMessageInputCursorDrawable="@color/yellow"
            android:visibility="gone"
            />
           
            
```

**Message List View**

```
    <io.getstream.chat.android.ui.message.list.MessageListView
        android:id="@+id/messageListView"
        android:layout_width="0dp"
        android:layout_height="0dp"
        android:layout_marginHorizontal="0dp"
        android:clipToPadding="false"
        android:background="@color/black"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintStart_toStartOf="parent"
        app:streamUiFlagMessageConfirmationEnabled="false"
        app:streamUiFlagMessageEnabled="false"
        app:streamUiMuteUserEnabled="false"
        app:streamUiPinMessageEnabled="false"
        app:streamUiEditMessageEnabled="false"
        app:streamUiMessageBackgroundColorMine="@color/purple"
        app:streamUiMessageBackgroundColorTheirs="@color/blue"
        app:streamUiMessageStrokeColorTheirs="@color/blue"
        app:streamUiMessageTextColorMine="@color/white"
        app:streamUiMessageTextColorTheirs="@color/black"
        app:streamUiDateSeparatorBackgroundColor="#7A7A7A"
        app:streamUiMessageTextColorDateSeparator="#FCFCFC"
        />

```
Regards,
Ramana.

**Reply by MarinTolic** [reply]
I am unable to reproduce the  state shown in the screenshot using the code you have provided to us.

In order to make it easier for us to diagnose and fix the potential issue, could you do one of the following:

1. Share a repository containing a running example of the state you've shown
2. Edit one of our sample apps in a manner which produces the state you've shown and provide a git patch to us

If you choose the second option, you have two ways of reproducing the state. One is by editing our [tutorial app code](https://github.com/GetStream/android-chat-tutorial), and the other by editing the UI Components sample app in our [main repository](https://github.com/GetStream/stream-chat-android/tree/main/stream-chat-android-ui-components-sample). 

I would advise you to edit the tutorial app as it is more faithful to the stock experience we provide, where as our main repo sample app has additional features which might cause side effects.

**Comment by ramanareddy463** [comment]
Thanks for the reply @MarinTolic 

I figured out the problem, Actually, the text color is taken from the app theme text color.

Regards,
Ramana

**Reply by MarinTolic** [reply]
Thanks for the reply @ramanareddy463 ,

We decided to expose styling options for this anyways so that you could have an easier time customizing this option if you need to. 

You can check the changes in the following PR: https://github.com/GetStream/stream-chat-android/issues/4676
Once it's merged it will be included in the next release.
